### PR TITLE
Render category search within a link on sidebar

### PR DESCRIPTION
**Pull request by adcpm**
This update for add the search as a link on sidebar. While typing for search a category, what you are typing will also appear in a link. To be able to browse category even if its not on the list.
image

### CODE REVIEW

**Comment by adcpm** [comment]
I've also fix the page /tags, it was not working anymore probably due on the Hard fork, the WS response change the naming from "categories" to "tags". And removed the triggers.

File: src/user/UserProfile.js [hunk 1]
```diff
@@ -134,7 +125,17 @@ class Profile extends Component {
                   : () => this.props.addUserFavorite(username)
                 }
               />
-              { has(jsonMetadata, 'profile.name') ? jsonMetadata.profile.name : `@${username} ` }
+              { has(jsonMetadata, 'profile.name') ? jsonMetadata.profile.name : `@${username}` }
+              { ' ' }
+              <Follow
```
Comment (nil1511) [inline src/user/UserProfile.js]: Follow does not uses username, params and edit properties.

File: src/app/Sidebar.js [hunk 2]
```diff
@@ -124,6 +124,24 @@ export default class Sidebar extends Component {
               #{category}
             </Link>
           </li>
+        );
+    }
+    return [];
+  }
+
+  renderSearchAsTag() {
+    const { search, categories } = this.state;
```
Comment (nil1511) [inline src/app/Sidebar.js]: `this.state.search` should be clear on onClickMenu.

File: src/user/UserProfile.js (hunk 1 above)
Comment (adcpm) [inline src/user/UserProfile.js]: Right, well see thank you!

**Pull request merged**